lib/chai/core/assertions.js

```diff
@@ -521,14 +522,26 @@ module.exports = function (chai, _) {
    * @alias greaterThan
    * @param {Number} value
    * @param {String} message _optional_
+   * @namespace BDD
    * @api public
    */
 
   function assertAbove (n, msg) {
     if (msg) flag(this, 'message', msg);
-    var obj = flag(this, 'object');
-    if (flag(this, 'doLength')) {
+    var obj = flag(this, 'object')
+      , doLength = flag(this, 'doLength');
+
+    if (doLength) {
       new Assertion(obj, msg).to.have.property('length');
+    } else {
+      new Assertion(obj, msg).is.a('number');
+    }
+
+    if (_.type(n) !== 'number') {
+      throw new Error('the argument to above must be a number');
```

Comment:
@meeber I'm not sure this sentence is clear enough, however, I'm not a native english speaker, would you mind suggesting a new message for this?
What do you think about: `The .above assertion must receive a Number as argument`?